### ISSUE TITLE
question about metric

### ISSUE THREAD

**Issue by liu-wei-song** [autonomousvision/navsim#76]
Hi

When I use navsim v2 to test transfuser: bash scripts/evaluation/run_transfuer.sh, I got the following result:

```
,token,valid,no_at_fault_collisions,drivable_area_compliance,driving_direction_compliance,traffic_light_compliance,ego_progress,time_to_collision_within_bound,lane_keeping,history_comfort,two_frame_extended_comfort,score
0,3bbe96de854e5ff3,True,1.0,1.0,1.0,1.0,1.0,1.0,1.0,0.0,0.0,0.0
1,42dc421e37c154e5,True,1.0,1.0,1.0,1.0,1.0,1.0,1.0,0.0,,0.0
2,63ec12363eef5e89,True,1.0,1.0,1.0,1.0,1.0,1.0,1.0,0.0,0.0,0.0
3,b2f5adcdf3dd5227,True,1.0,1.0,1.0,1.0,1.0,1.0,1.0,0.0,0.0,0.0
4,0503cc398e2656be,True,1.0,1.0,1.0,1.0,1.0,1.0,1.0,0.0,,0.0
```

The contents of the sh file are as follows:

```
TRAIN_TEST_SPLIT=navtest
CHECKPOINT=navsim_baselines/transfuser/transfuser_seed_0.ckpt

python $NAVSIM_DEVKIT_ROOT/navsim/planning/script/run_pdm_score.py \
train_test_split=$TRAIN_TEST_SPLIT \
agent=transfuser_agent \
agent.checkpoint_path=$CHECKPOINT \
experiment_name=transfuser_agent_eval
```

Are there any suggestions that could help me further identify the cause?

**Comment by VVeiCao** [comment]
Hi @liu-wei-song，

In Navsim2.0, we currently do not provide Lidar data, so Lidar-based models like Transfuser are not supported in this version. However, feel free to try the latent Transfuser.

Best,
Wei

**Comment by liu-wei-song** [comment]
thanks
